src/Notes.cc

```diff
@@ -107,11 +109,12 @@ Note::updateNotePairs(NotePairs::Pointer pairs, const CharacterSet *delimiters,
 void
 Note::dump(StoreEntry *entry, const char *k)
 {
+    PackableStream out(*entry);
     for (const auto &v: values) {
-        storeAppendPrintf(entry, "%s %.*s %s",
-                          k, key().length(), key().rawContent(), ConfigParser::QuoteString(SBufToString(v->value())));
+        out << k << ' ' << key() << ' ' << Format::DquoteString(v->value());
```

Comment:
FYI, this is not correct way to format "key=value" (missing '='). Have left this for bug-compatibility reasons.

Reply:
Hmm. okay. that is missing documentation then.